### DISCUSSION TITLE
when delete photo, the app asks confirmatio twice.

### DISCUSSION THREAD

**Discussion by adarshkannada** [IacobIonut01/Gallery#527] [General]
on verion 3.0.1 latest from github.
android 14 latest build, pixel 8.

when i try to delete a photo or video the app asks confirmation two times.
i want it to delete even without a single confirmation after i press delete button.
how can this be done?

**Comment by mcbabo** [comment]
Late but nontheless, 
You have to give the app full access to all files, not just storage scopes.
